src/idvc/dvc_interface.py

```diff
@@ -1868,6 +1863,16 @@ def CreateMaskPanel(self):
         formLayout.setWidget(widgetno,QFormLayout.FieldRole, mp_widgets['extendMaskCheck'])
         widgetno += 1
 
+        # Add start tracing button
+        mp_widgets['start_tracing'] = QPushButton(groupBox)
+        mp_widgets['start_tracing'].setText("Start Tracing")
+        mp_widgets['start_tracing'].clicked.connect(self.ToggleTracing)
+        mp_widgets['start_tracing'].setCheckable(True)
+        mp_widgets['start_tracing'].setChecked(False)
+        # mp_widgets['start_tracing'].setToolTip("Press 't' and draw a region on the viewer to create a mask.")
```

Comment:
could we set a tooltip to be consistent with other buttons and gui elements, perhaps:
```suggestion
        mp_widgets['start_tracing'].setToolTip("Begin drawing a region on the viewer to create a mask.")
```